src/app/Scenes/Feature/Feature.tests.tsx

```diff
@@ -4,7 +4,7 @@ import { renderWithWrappersLEGACY } from "app/utils/tests/renderWithWrappers"
 import { createMockEnvironment, MockPayloadGenerator } from "relay-test-utils"
 import { FeatureQueryRenderer } from "./Feature"
 
-let mockRelayEnvironment = defaultEnvironment as ReturnType<typeof createMockEnvironment>
+let mockRelayEnvironment = defaultEnvironment as unknown as ReturnType<typeof createMockEnvironment>
```

Comment:
👀 Would this work as well?

```suggestion
let mockRelayEnvironment = defaultEnvironment as unknown
```

Reply:
nope, it will raise a type error when trying to access `mockRelayEnvironment.mock.resolveMostRecentOperation`

Reply:
What is `as unknown as ReturnType<typeof createMockEnvironment>` actually doing?

Reply:
when casting type A to B typescript checks if type A properties are include in type B, if not it raises an error. in this case, I am confident the types are accurate regardless of the typescript warning. Typescript docs suggest in this case to always casting as `unknown` the to the type you need (instead of casting as `any`)